groceries_app/models.py

```diff
@@ -6,9 +6,9 @@ class Meal(models.Model):
     name = models.CharField(max_length=60, unique=True)
     recipe_category = models.CharField(max_length=40)
     times_eaten = models.PositiveIntegerField(default=0)
-    cook_time = models.PositiveIntegerField(null=True, blank=True)
+    cook_time_in_minutes = models.PositiveIntegerField(null=True, blank=True)
     portion_number = models.PositiveIntegerField(null=True, blank=True)
-    photo = models.ImageField(null=True, blank=True)
+    photo = models.ImageField(null=True, blank=True, upload_to="gallery")
```

Comment:
Is `gallery` just a folder name or a Django specific thing?

Reply:
Gallery is just a folder name that is in the media root. It was the name used in the tutorial I working through so I named it accordingly just to see if it would work. This can probably be named to something more descriptive.